docs/checks/internal/checks.yaml

```diff
@@ -199,7 +199,8 @@ checks:
         - Prevent branch deletion
 
       Tier 2 Requirements (6/10 points):
-        - Require at least 1 reviewer for approval before merging
+        - For administrators: Require PRs prior to make any code changes
+        - Require at least 1 reviewer for approval before merging (for administrators, this requirement weights twice than the others in this tier)
```

Comment:
nit: move the require PR text down to keep all of the for admininstrators: together